contracts/DCAHubSwapper/DCAHubSwapperParameters.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '../interfaces/IDCAHubSwapper.sol';
+import '../utils/Governable.sol';
+
+abstract contract DCAHubSwapperParameters is Governable, IDCAHubSwapperParameters {
```

Comment:
Personal and ignorable opinion: I think it would make more sense to move this to the SwapHandler. This extra contract is not needed, and it doesn't add much.